Simplified/NYPLAccountSignInViewController.m

```diff
@@ -621,6 +621,7 @@ - (void)showEULA
 - (void)updateLoginLogoutCellAppearance
 {
   if (self.isCurrentlySigningIn) {
+    self.eulaCell.userInteractionEnabled = NO;
```

Comment:
I can't say I took the time to track this down fully, but why do we need this when we are also apparently calling `[UIApplication beginIgnoringInteractionEvents]`?